2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,939 @@
+			</section>
+
+			<section>
+				<h1>Übung</h1>
+				<p><span class="fragment">z.B. Titel, Verlag, Erscheinungsort und -jahr aus den PICA-Feldern 021A.a, 033A.n, 033A.p, 011@.n verwenden</span></p>
+				<p><span class="fragment">Verlag und Erscheinungsort sollen in einem neuen Feld kombiniert werden</span></p>
+				<p><span class="fragment">Als Feldnamen wollen wir sprechende, nicht-numerische Bezeichnungen haben</span></p>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-fix#functions-and-cookbook">z.B. mit move_field, paste, retain, s. Dokumentation</a></span></p>
+			</section>
+
+			<section>
+				<h2>move_field, paste, retain</h2>
+				<span class="fragment">
+					<pre><code class="nohighlight" data-trim>
+1{a: Faust, b {n: Goethe, v: JW}, c: Weimar}
+2{a: Räuber, b {n: Schiller, v: F}, c: Weimar}
+					</code></pre>
+				</span>
+				<span class="fragment">
+					<pre><code class="nohighlight" data-trim>
+move_field(a, title)
+paste(author, b.v, b.n, '~aus', c)
+retain(title, author)
```

Comment:
```suggestion
move_field("a", "title")
paste(author, "b.v", "b.n", "~aus", "c")
retain("title", "author")
```

Hier zum Beispiel: 
https://github.com/hbz/slides/pull/79/files/2f799ec84710114cb45d7984a0aafcadff2d9a05#r878052843

Reply:
Da wir sonst meist einfache Anführungszeichen haben würde das hier alleine es nicht konsistenter machen, s. auch https://github.com/hbz/slides/pull/79#discussion_r878171504.